### PR TITLE
Update 096 TC 

### PR DESCRIPTION
**Pull request by X-EROS**
# SCP:SL Translation PR Template

## Language Name
zh-Hant

## State Type of change (New language, changed existing Etc.)
Update

# Checklist:
- [ ] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation

### CODE REVIEW

File: Translations/zh-Hant/Abilities.txt [hunk 1]
```diff
@@ -12,9 +12,9 @@
 腐蝕：你能夠穿過所有已關上的門。
 天鵝之歌：除了手榴彈和電擊之外，你幾乎不受任何子彈傷害，但你的收容室有一個小房間，任何玩家都可以犧牲自己。當有人犧牲後，任何人都可以再按下那個房間裡的一個按鈕，你將會即刻回到那個小房間被重新收容。
 次元袋：捕獲人類並將其傳送到有八道門的次元袋中，其中一道門會將他們傳送到設施內的隨機一個房間，其餘的七道門則會殺死他們。\ n人類會在這個空間中慢慢受到侵蝕而死去。
-害羞：在普通模式下，你的移速緩慢而且無法攻擊，並靜靜地發出哭聲。 但如果有人類長時間看著你，你將會站在原地五秒鐘，然後進入憤怒模式。
-反社會之怒：在憤怒模式下，你將獲得大量移速加成並且大聲尖叫。 憤怒模式持續時間約二十秒，在效果結束後你不能在短時間內再次觸發憤怒模式，在此期間你不會再哭。\ n在憤怒模式下，按下或按住左鍵會立即秒殺攻擊範圍內的人類。
-強破：在憤怒模式下，與門有互動時會破壞該門。
+注視恐懼症：大多數時候你將處於平靜而溫和 - 但你不喜歡被人傷害或被看到你的臉。一旦受到傷害或者被看到，你得殺死這些該死的罪人來平息你的憤怒。在你的憤怒消退之前，跟隨著他們的光芒。 每位無禮的罪人都會提高您的 AHP，從而減少在你站立時所受到的傷害。
+欲哭無淚 (溫和狀態)：這實在太難了。倚靠在門上或許有幫助 ; 您將會變得更安靜，站在那裡讓自己平靜下來……或等待某位的來臨。
+近身攻擊 (憤怒狀態)：在一般情況下，你得要保持理智。 但是，當您被激怒時……您會撕破任何您觸碰的人。 當您揮擊時，你將毀滅前方的區域中的門和肉體。
```
Comment (Tychus-Xiaoyi) [inline Translations/zh-Hant/Abilities.txt:17]: You've made a little mistake in your translation.句子最後部分我們想描述的是重做後的096能夠撕開重型閘門和蓄力遠眺的能力，希望您可以及時修改一下❤

Comment (X-EROS) [reply Translations/zh-Hant/Abilities.txt:17]: 蓄力遠眺是什麼?

Comment (Tychus-Xiaoyi) [reply Translations/zh-Hant/Abilities.txt:17]: 重做後的096技能，詳情請參考https://youtu.be/cPF-FlEE_uo的1:06處

Comment (Tychus-Xiaoyi) [reply Translations/zh-Hant/Abilities.txt:17]: 參考https://youtu.be/cPF-FlEE_uo 的1:06處，這裡的swings還有飛撲的含義。
```suggestion
近身攻擊 (憤怒狀態)：在一般情況下，你得要保持理智。 但是，當您被激怒時……您會撕破任何您觸碰的人。 當您揮擊時，你將毀滅前方的區域中的門和肉體。

```

Comment (X-EROS) [reply Translations/zh-Hant/Abilities.txt]: 改好了

**Pull request merged**